src/static/js/modules/pagination-validation.js

```diff
@@ -0,0 +1,50 @@
+/* ==========================================================================
+   Pagination Form validation
+   Check to verify page input value is number and within min/max range
+   before submitting form.
+   ========================================================================== */
+
+'use strict';
+
+// Import modules
+var $ = require( 'jquery' );
+
+/**
+ * Set up DOM references and event handlers.
+ */
+function init() {
+  $( '.js-validate_pagination' ).each( function() {
+    var $form = $( this );
+    var $placeHolder = $( '<div class=\'cf-notification__pagination-ctr\'></div>' );
+    $form.parents( 'nav.pagination' ).before( $placeHolder );
+    $( $placeHolder ).cf_notifier();
```

Comment:
On second thought, why not move this all into `form-validation.js`? I set that up to watch all forms, so you could easily extend it to send pagination messages as well.